inno_setup/gnucash-mingw64.iss

```diff
@@ -41,17 +41,17 @@ Name: desktopicon; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm
 Name: menuicon; Description: "{cm:CreateMenuLink}"; GroupDescription: "{cm:AdditionalIcons}"
 
 [Icons]
-Name: "{group}\GnuCash"; Filename: "{app}\bin\@PACKAGE@.exe"; WorkingDir: "%USERDOCS%"; Comment: "{cm:IconComment_GnuCash}"; IconFilename: "{app}\share\@PACKAGE@\pixmaps\gnucash-icon.ico"; Tasks: menuicon
+Name: "{group}\GnuCash"; Filename: "{app}\bin\@PACKAGE@.exe"; WorkingDir: "{code:GetDocPath}"; Comment: "{cm:IconComment_GnuCash}"; IconFilename: "{app}\share\@PACKAGE@\pixmaps\gnucash-icon.ico"; Tasks: menuicon
```

Comment:
What's the problem with %USERDOCS% ? That's not documented here.

Reply:
There's no such variable.

Reply:
Hmm, do you remember why you originally changed to `%USERDOCS%` in the first place ? Was the GetDocPath variant causing issues ?

Reply:
Note that unlike what your original commit to change from GetDocPath to USERDOCS suggests, this directory is used a runtime not at install time.
Setting this properly to point at the user's "My Documents" directory ensures the very first call to `File->Save` on a clean installation offers `My Documents` rather than gnucash' installation directory to save to. Same for `File->Open`.

Reply:
As I said in my first comment it was probably a test commit that got swept up with 5d6f7db24d46e91f168855ce20308f4f42878e1d.

Reply:
Okay. I didn't connect those dots...

So this changes my question slightly: besides the USERDOCS change 1a89d04 also inserts a cmake_toolchain. Was that also unintentional ? If so, I'd rather see the reversion commit generated using `git revert` instead of manually redoing the changes. `git revert` generates a commit message that clearly indicates which commit got reverted, making future archeology on the git repo easier. Ideally the reason of the reversal should be added in the commit message during the command run.

Reply:
Yes, it was a start to an experiment that never went anywhere. I agree about the revert commit and I just pushed one.